### ISSUE TITLE
requirements.txt install fails with numpy.distutils.core import error

### ISSUE THREAD

**Issue by fbessho** [fbessho/PyPoi#23]
`pip install -r requirements.txt` fails with the error  on Max OS X.

```
pip install -r requirements.txt

...
    
        from numpy.distutils.core import setup
    
    ImportError: No module named numpy.distutils.core
    
    ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/3s/7v9d19q90752p8_nqzxj4cg80000gn/T/pip-build-MKi69E/pyamg
```

Current workaround is to install numpy independently before running `pip install -r requirements.txt`

```
pip install numpy==1.8.1
```

**Comment by fbessho** [comment]
Related issue in github
https://github.com/scipy/scipy/issues/3379
https://github.com/scipy/scipy/pull/465

**Comment by fbessho** [comment]
Kinda resolved by 0.2.1 release (#24)
`pip install` should work without this issue.
